crescent/commands/options.py

```diff
@@ -115,7 +115,7 @@ def __get__(self: Self, inst: None, cls: Any) -> Self:
         ...
 
     @overload
-    def __get__(self: Self, inst: object, cls: Any) -> T:
+    def __get__(self, inst: object, cls: Any) -> T:
```

Comment:
The typed self with `TypeVar("Self")` can be removed here, correct?

Reply:
yes